code/modules/mob/interactive.dm

```diff
@@ -119,29 +119,34 @@
 		retal_target = potentialAssault
 	..()
 
-/mob/living/carbon/human/interactive/verb/customiseSNPC()
+/client/proc/customiseSNPC(var/mob/A in world)

```

Comment:
I can't remember whether BYOND had a hack or not that makes this actually faster for verbs.